rig/machine_control/machine_controller.py

```diff
@@ -764,10 +770,10 @@ def _send_ffd(self, pid, aplx_data, address):
             address += data_size
             pos += data_size
 
-    def _send_ffe(self, pid, app_id, app_flags, cores, fr):
+    def _send_ffe(self, pid, app_id, app_flags, fr):
         """Send a flood-fill end packet."""
         arg1 = (NNCommands.flood_fill_end << 24) | pid
-        arg2 = (app_id << 24) | (app_flags << 18) | (cores & 0x3ffff)
+        arg2 = (app_id << 24) | (app_flags << 18)
```

Comment:
Worth a comment highlighting that the core set is specified by `_send_ffcs` (since looking at the scamp code may not make this obvious)?